pkg/operator/ceph/csi/spec.go

```diff
@@ -511,6 +556,14 @@ func createCSIDriverInfo(clientset kubernetes.Interface, name string, ownerRef *
 	return err
 }
 
+// deleteCSIDriverInfo Registers CSI driver by creating a CSIDriver object
+func deleteCSIDriverInfo(clientset kubernetes.Interface, name string) error {
+	_ = func(options *metav1.DeleteOptions) error {
```

Comment:
No need for the embedded func here either. Why not just delete the csi driver directly?

Reply:
Done.